### DISCUSSION TITLE
How to study python

### DISCUSSION THREAD

**Discussion by icrushhaileesteinfeld** [codewars/codewars.com#2823] [General]
i am new to python

**Comment by CodeByAidan** [comment]
It would be helpful to provide more detail/information on your question? (i.e. studying using a book, studying online), there's many ways to describe this question more in detail.

**Reply by icrushhaileesteinfeld** [reply]
i just start studying python today, i never coding before, so how can i start to self study, and what resourses i can use

**Reply by icrushhaileesteinfeld** [reply]
i notice that codewars is just a place to practice python, so bruh

**Reply by z-b-b** [reply]
Hi, you can start learning python through this website >> 
https://www.w3schools.com/python/default.asp

**Reply by icrushhaileesteinfeld** [reply]
tks alot!

**Reply by zol-dev** [reply]
هيوو مافي زيو المهم انا اليوم دحومي XXXXXXX LUGDKH

**Reply by CodeByAidan** [reply]
> هيوو مافي زيو المهم انا اليوم دحومي XXXXXXX LUGDKH

????/

**Comment by ghost** [comment]


**Reply by CodeByAidan** [reply]
Also @icrushhaileesteinfeld ...
![image](https://user-images.githubusercontent.com/67598470/201114717-6aa6f167-3e12-4a83-8db5-940ca4c83208.png)
